### PR TITLE
delete comment logic

### PR DESCRIPTION
**Pull request by Gusvarov**

### CODE REVIEW

File: src/app/component/comments/components/comment-body/comment-body.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="comment-body-wrapper" *ngFor="let element of elements">
+<div class="comment-body-wrapper" *ngFor="let element of elements; let i = index">
```
Comment (KimAT25) [inline src/app/component/comments/components/comment-body/comment-body.component.html:1]: fix alignment

File: src/app/component/comments/components/delete-comment/delete-comment.component.ts [hunk 2]
```diff
@@ -1,17 +1,30 @@
...
 
-  ngOnInit() {
+  @Input() public element;
+  @Input() public elements;
+
+  public deleteComment(): void {
+    this.commentsService.deleteComments(this.element.id).subscribe(response => {
+      if (response.status === 200) {
+        this.elements = this.elements.filter((item, index) => {
+           if (item.text === this.element.text) {
```
Comment (KimAT25) [inline src/app/component/comments/components/delete-comment/delete-comment.component.ts:22]: please try to use ternary operator for line 20 and 22

File: src/app/component/comments/services/comments.service.ts [hunk 3]
```diff
@@ -29,4 +29,8 @@ export class CommentsService {
   public getCommentsByPage(): Observable<object> {
     return this.http.get(`https://greencity.azurewebsites.net/econews/comments?ecoNewsId=${this.ecoNewsId}&page=0&size=12`);
   }
+
+  public deleteComments(id) {
+    return this.http.delete(`https://greencity.azurewebsites.net/econews/comments?id=${id}`, {observe: 'response'});
```
Comment (KimAT25) [inline src/app/component/comments/services/comments.service.ts:34]: Please use alias for part of link line 34. We have this alias.

File: src/app/component/comments/components/comment-body/comment-body.component.html (hunk 1 above)
Comment (Gusvarov) [reply src/app/component/comments/components/comment-body/comment-body.component.html]: fixed

File: src/app/component/comments/services/comments.service.ts (hunk 3 above)
Comment (Gusvarov) [reply src/app/component/comments/services/comments.service.ts]: fixed

**Pull request merged**